src/scripts/components/layers/time-slider/time-slider.tsx

```diff
@@ -79,6 +87,12 @@ const TimeSlider: FunctionComponent = () => {
     []
   );
 
+  // clamp globe time to min/max of the active layers when a layer changes and stop playback
+  useEffect(() => {
```

Comment:
Don't we have the combined time ranges with a min/max value already in the timeslider? Why the extra action and not just call `setGlobeTime(clampToRange(time, combined.min, combined.max))` directly?

Reply:
yep, correct. Will change that.